packages/local-workspace/test/workspace.test.ts

```diff
@@ -280,17 +280,17 @@ describe('local workspace', () => {
           })
         })
         it('should use the credentials source that was passed as a paramter', async () => {
-          expect(mockLoad).toHaveBeenCalledWith(
-            expect.anything(),
-            expect.anything(),
-            credentialSource,
-            expect.anything(),
-            expect.anything(),
-            expect.anything(),
-            expect.anything(),
-            undefined,
-            expect.anything(),
-          )
+          expect(mockLoad).toHaveBeenCalledWith({
+            config: expect.anything(),
+            adaptersConfig: expect.anything(),
+            credentials: credentialSource,
+            environmentsSources: expect.anything(),
+            remoteMapCreator: expect.anything(),
+            ignoreFileChanges: expect.anything(),
+            persistent: expect.anything(),
+            getCustomReferences: expect.anything(),
+            adapterCreators: expect.anything(),
+          })
```

Comment:
nit - instead of having a bunch of `expect.anything`, if we only care about the credentialSource, I think this can be changed to
```suggestion
          expect(mockLoad).toHaveBeenCalledWith(expect.objectContaining({
            credentials: credentialSource,
          }))
```